projects/canopy/src/lib/carousel/carousel.component.ts

```diff
@@ -41,6 +41,14 @@ export class LgCarouselComponent implements AfterContentInit, OnDestroy {
   pause = new BehaviorSubject<boolean>(false);
   pausableTimer$: Observable<void>;
 
+  pauseCarousel() {
+    this.pause.next(true);
+  }
+
+  playCarousel() {
```

Comment:
Where are you using these functions? are they not supposed to be linked to the template?

Reply:
Yes, they are in template. I think I resolved the conflict and methods got removed. Will update the PR.

Reply:
These 2 methods Play and Pause carousel will be exposed by targeting the carousel with ViewChild decorators in the application(OSS) to fix the accessibility issues.